src/features/emails/components/EmailEditor/tools/paragraphWithSpanPaste.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+//@ts-ignore
+import Paragraph from '@editorjs/paragraph';
+import { BlockTool, HTMLPasteEvent } from '@editorjs/editorjs';
+
+//@ts-ignore
+export default class ParagraphWithSpanPaste
+  extends Paragraph
+  implements BlockTool
+{
+  onPaste(event: HTMLPasteEvent) {
+    const text = event.detail.data.textContent;
+    event.detail.data = document.createElement('div');
+    event.detail.data.textContent = text;
+    return super.onPaste(event);
+  }
+
+  static get pasteConfig() {
+    return {
+      tags: ['P', 'SPAN'],
```

Comment:
I'm realizing now that this filter is probably the reason why we only got `span`-related events in `onPaste()`. I wonder if there is some sort of wildcard that we could use, but for now, this is good enough because I believe it will solve the bug!